### PR TITLE
[VEN-982] Increase test coverage

### PR DESCRIPTION
**Pull request by UrAvgDeveloper**
## Description

This PR increase coverage of the following contracts.

- Comptroller
- vToken
- shortfall
- accessControlManager
- PoolLens

Resolves [VEN-982](https://jira.toolsfdg.net/browse/VEN-982)

## Checklist
- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: tests/hardhat/Comptroller/setters.ts [hunk 1]
```diff
@@ -115,4 +133,62 @@ describe("setters", async () => {
...
+
+    it("reverts if liquidation threshold is greater then collateral factor", async () => {
+      await comptroller.connect(poolRegistrySigner).supportMarket(OMG.address);
+      await expect(
+        comptroller.setCollateralFactor(OMG.address, convertToUnit("0.7", 18), convertToUnit("0.8", 18)),
+      ).to.be.revertedWithCustomError(comptroller, "InvalidLiquidationThreshold");
+    });
+
+    it("reverts if token price is zero", async () => {
+      await oracle.getUnderlyingPrice.returns(0);
```
Comment (0xlucian) [inline tests/hardhat/Comptroller/setters.ts:187]:
```suggestion
      oracle.getUnderlyingPrice.returns(0);
```
no need to await the faking of return value of a fake contract

File: tests/hardhat/Comptroller/setters.ts [hunk 2]
```diff
@@ -44,12 +46,21 @@ describe("setters", async () => {
   let user: SignerWithAddress;
   let accessControl: FakeContract<AccessControlManager>;
   let comptroller: MockContract<Comptroller>;
+  let OMG: FakeContract<VToken>;
+  let poolRegistry;
+  let oracle;
```
Comment (0xlucian) [inline tests/hardhat/Comptroller/setters.ts:51]:
```suggestion
  let poolRegistry: FakeContract<PoolRegistry>;
  let oracle: FakeContract<PoolRegistry>;
```

**Review by 0xlucian — APPROVED** [review]
LGTM overall. Only some nitpicks

File: tests/hardhat/Comptroller/setters.ts (hunk 1 above)
Comment (UrAvgDeveloper) [reply tests/hardhat/Comptroller/setters.ts]: Fixed

File: tests/hardhat/Comptroller/setters.ts (hunk 2 above)
Comment (UrAvgDeveloper) [reply tests/hardhat/Comptroller/setters.ts]: Fixed

**Review by chechu — APPROVED** [review]

**Review by 0xlucian — APPROVED** [review]

**Review by defcon022 — APPROVED** [review]
LGTM

**Pull request merged**